### PR TITLE
fix demo simple interface, issues with imports, bump to 1.0.2

### PR DESCRIPTION
**Pull request by WilsonGregory**
## Changes
- fix imports in other_models, vae_models
- simplify imports in DemoSimpleInterface notebook now that its on pip
- fix an issue in train model where if the validation set is empty, EarlyStopping on val_loss causes issues. So now it only does early stopping if there is a validation set

## Testing
- Ran demo simple interface
- ran a few other notebooks that I have locally

### CODE REVIEW

File: notebooks/DemoSimpleInterface.ipynb [hunk 1]
```diff
@@ -37,12 +37,7 @@
    },
    "outputs": [],
    "source": [
-    "!pip install pytorch-lightning==1.3.3\n",
-    "# !pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.8.1-cp37-cp37m-linux_x86_64.whl\n",
-    "!pip install scGeneFit\n",
-    "!pip install scanpy\n",
-    "!pip install anndata\n",
-    "!pip install lassonet"
+    "!pip install markermap"
```
Comment (beelze-b) [inline notebooks/DemoSimpleInterface.ipynb:40]: Installing our package with pip will automatically install these as dependencies? Really cool!

File: notebooks/DemoSimpleInterface.ipynb [hunk 2]
```diff
@@ -108,8 +89,10 @@
     "from torch.autograd import Variable\n",
     "from torch.nn import functional as F\n",
     "\n",
-    "from utils import *\n",
-    "from sklearn.preprocessing import LabelEncoder\n"
+    "from markermap.vae_models import MarkerMap\n",
+    "from markermap.utils import get_zeisel\n",
```
Comment (beelze-b) [inline notebooks/DemoSimpleInterface.ipynb:93]: what is get_zeisel? Is it from our package? I wonder if it should be something like MarkerMap.get_zeisel if possible.

**Comment by beelze-b** [comment]
Looks mostly alright to me. I just had two questions.

File: notebooks/DemoSimpleInterface.ipynb (hunk 1 above)
Comment (WilsonGregory) [reply notebooks/DemoSimpleInterface.ipynb:40]: Yeah, it will install all the packages in the `install_requires` field of setup.cfg: https://github.com/Computational-Morphogenomics-Group/MarkerMap/blob/6c7efcaea60180a04f5cac80ef5925d93e8f0a58/setup.cfg

In the future, we could split the package so that you can download a version that doesn't include all the other models (LassoNet, RankCorr, and Smashpy), so there would be a lot less dependencies if you just wanted to run MarkerMap.

File: notebooks/DemoSimpleInterface.ipynb (hunk 2 above)
Comment (WilsonGregory) [reply notebooks/DemoSimpleInterface.ipynb:93]: Yeah, `get_zeisel` is one of 4 data import and preprocessing functions that I wrote just because I was writing the same code over and over again. The others are `get_citeseq`, `get_mouse_brain`, and `get_paul`. I will change this import to make it more clear when you are using the function.

**Review by beelze-b — APPROVED** [review]
